src/Orleans/LogConsistency/ILogConsistencyProvider.cs

```diff
@@ -30,14 +30,20 @@ public interface ILogViewAdaptorFactory
         /// <summary> Returns true if a storage provider is required for constructing adaptors. </summary>
         bool UsesStorageProvider { get; }
 
+
+        /// <summary> Returns true if a event-storage provider is required for constructing adaptors. </summary>
+        bool UsesEventStorageProvider { get; }
```

Comment:
This & `UsesStorageProvider` above appear to be a candidates for activation-scoped dependency injection via facets. @jdom, what do you think?